### ISSUE TITLE
how come that it don't find any on my device

### ISSUE THREAD

**Issue by JaPro1005** [DavidMGilbert/btc-hack#74]
it has checkt over 100000 and still didn't found any does that mean that it don't work on windows or do i need ckeck more?

**Comment by MvdS74** [comment]
> it has checkt over 100000 and still didn't found any does that mean that it don't work on windows or do i need ckeck more?

Nah, the script just doesn't work for more than one reason. The used API is limited to 100 calls per hour. So roughly once per 36 seconds. This script gets you blocked from the API in seconds but it doesn't report that. It pretends to be checking but it can't.

And even if you use a faster API, almost all of the mnemonic phrases it generates are invalid (The mnemonic variant) and the algorithm to generate the wallet addresses just doesn't work. All 'addresses' that the scripts check are just garbage.

**Comment by MvdS74** [comment]
> it has checkt over 100000 and still didn't found any does that mean that it don't work on windows or do i need ckeck more?

Oh, and even if the scripts would work, with 2<sup>190</sup> possibilities, the chances to find a funded wallet are (almost) non existant. So you could easily be checking quazillions of (valid) addresses and still find nothing
